en/docs/SetupAndInstall/DeployingWSO2APIManager/configure_apim_analytics/configure_worker/active_active.md

```diff
@@ -0,0 +1,135 @@
+# Configure a Analytics workers as active-active
+
+To configure API-M Analytics as a scalable deployment, you can use the Active-Active deployment pattern. For an 
+overview of the Active-Active deployment pattern and instructions to configure it, see the following topics.
+
+- [Overview](#overview)
+- [Configuring an active-active cluster](#configuring-an-active-active-cluster)
+
+
+## Overview
+![](../../../../assets/img/setup-and-install/configure_apim_analytics/Active-Active_Deployment.png)
```

Comment:
Use {{base_path}} in the URLs without ../

Example:
{{base_path}} /assets/img/setup-and-install/configure_apim_analytics/Active-Active_Deployment.png

Reply:
Fixed